### PR TITLE
fix(w3c trace context): do not pass down unknown flags.

### PR DESCRIPTION
**Pull request by basti1302**
The W3C trace context specification mandates that a participant must only send known flags downstream. See
https://www.w3.org/TR/trace-context/#other-flags: "The behavior of other flags, such as (00000100) is not defined and is reserved for future use. Vendors MUST set those to zero."

### CODE REVIEW

**Comment by Ferenc-** [comment]
@basti1302 Rebasing onto the current `master` branch should unblock you.

**Review by pvital — APPROVED** [review]
Looks good. Great job.

**Comment by pvital** [comment]
@basti1302 this PR needs to rebase before merging it.

**Comment by basti1302** [comment]
> @basti1302 this PR needs to rebase before merging it.

@pvital I just rebased again, should be good now.

**Comment by Ferenc-** [comment]
> > @basti1302 this PR needs to rebase before merging it.
> 
> @pvital I just rebased again, should be good now.

My apologies, https://github.com/instana/python-sensor/pull/421 went in the meantime.

**Comment by pvital** [comment]
> > > @basti1302 this PR needs to rebase before merging it.
> > 
> > 
> > @pvital I just rebased again, should be good now.
> 
> My apologies, #421 went in the meantime.

@basti1302 if the PR is OK, we can rebase before merging it. Just say, and we can do it.

**Comment by basti1302** [comment]
> > My apologies, #421 went in the meantime.
> 
> @basti1302 if the PR is OK, we can rebase before merging it. Just say, and we can do it.

Lolol, no merges since weeks and today you are merging _all the PRs_ 🎉   No worries, I rebased again. But if you plan to merge more PRs before this one, you can also feel free to rebase this on your own.

**Pull request merged**
